### DISCUSSION TITLE
Divider/Separator

### DISCUSSION THREAD

**Discussion by not-yuvi** [OliveTin/OliveTin#389] [Ideas]
A simple divider/separator would be nice, similar to a `<hr>` tag. I have some things related to music (one of the examples) which would look really nice grouped together.

**Comment by jamesread** [comment]
Hey @not-yuvi , very keen to explore the idea more. Could you help me by visualizing or more describing how the divider would work? Could you throw together an example using MS Paint?

**Reply by jamesread** [reply]
Also, just checking you saw Dashboards; https://docs.olivetin.app/dashboards.html

**Reply by not-yuvi** [reply]
heres a figma concept that ive made (ignore the messy components lol): https://www.figma.com/proto/1TqWS5uetFyghJsFjT1K1Y/Untitled?node-id=0-1&t=KxWOeE6GoUUv0kBe-1

i guess these could be stated in the yaml file like this:
```
- title: Name
  icon: 😁
  shell: cmd

- title: Name
  icon: 😁
  shell: cmd

- [Music] -

- title: Music Command Name
  icon: 🎶
  shell: cmd

**Reply by jamesread** [reply]
Yeah, this is fieldsets which already exist in OliveTin: https://docs.olivetin.app/fieldsets.html
